### PR TITLE
Agregar al Gran Hermano

### PR DESCRIPTION
**Pull request by christophavila**
Agregar: espanol/granhermano/granhermano.md

### CODE REVIEW

**Comment by morosebose** [comment]
@casey-collab

File: espanol/granhermano/granhermano.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+La guerra es la paz. La libertad es la esclavitud. La ignorancia es la fuerza 
+
+Lo más característico de la vida moderna no era su crueldad ni su inseguridad, sino sencillamente su vaciedad, su absoluta falta de contenido
```
Comment (casey-collab) [inline espanol/granhermano/granhermano.md]: This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.

File: espanol/granhermano/granhermano.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+La guerra es la paz. La libertad es la esclavitud. La ignorancia es la fuerza 
+
+Lo más característico de la vida moderna no era su crueldad ni su inseguridad, sino sencillamente su vaciedad, su absoluta falta de contenido 
+
+Pero ya todo estaba arreglado, todo alcanzaba la perfección, la lucha había terminado. Se había vencido así mismo definitivamente. Amaba al Gran Hermano
```
Comment (casey-collab) [inline espanol/granhermano/granhermano.md]: This line is also too long.

File: espanol/granhermano/granhermano.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+La guerra es la paz. La libertad es la esclavitud. La ignorancia es la fuerza
```
Comment (casey-collab) [inline espanol/granhermano/granhermano.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: espanol/granhermano/granhermano.md (hunk 3 above)
Comment (christophavila) [inline espanol/granhermano/granhermano.md]: I have made the fixes in my local master, should I push the master now?

Christopher

**Comment by morosebose** [comment]
@casey-collab please review

File: espanol/granhermano/granhermano.md (hunk 1 above)
Comment (casey-collab) [inline espanol/granhermano/granhermano.md]: This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.

File: espanol/granhermano/granhermano.md (hunk 2 above)
Comment (casey-collab) [inline espanol/granhermano/granhermano.md]: This line is also too long.

File: espanol/granhermano/granhermano.md (hunk 3 above)
Comment (casey-collab) [inline espanol/granhermano/granhermano.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: espanol/granhermano/granhermano.md (hunk 3 above)
Comment (christophavila) [inline espanol/granhermano/granhermano.md]: Hi Casey

I have made the fixes in my **local** master, should I push my master now in order to update the main one?  "_git push origin master_"

Or do I need to create another pull request?

Thanks

Christopher

**Comment by morosebose** [comment]
@christophavila yes, please push the changes from your local master up to GitHub. Then Casey can act on the pull request :+1: sorry I misunderstood earlier!

**Comment by christophavila** [comment]
Done, I hace fixed the long lines in the first commit and the link to the new file in the second commit.

Thanks, Christopher :)

**Comment by morosebose** [comment]
@christophavila sadly now there is a merge conflict, as between the time of your original pull from the repo and your latest update to your fork, updates have been made to the master in Udacity's repo. Sorry :-(  Please resolve the merge conflict and try again

File: espanol/canudos.md [hunk 4]
```diff
@@ -93,7 +93,7 @@ and doesn't know how to speak Spanish.](../english/TheEpicSax/111_TheEpicSaxGuyA
 
 Al ojearlo, una duda lo emabargó...[¿Qué hago?](duda/duda.md)
 
-Al despertar y enfocar la vista vió lo que había sucedido tras la 
+Al despertar y enfocar la vista vió lo que había sucedido tras la
```
Comment (christophavila) [inline espanol/canudos.md]: Removing white space at the end of this line

Comment (christophavila) [inline espanol/canudos.md]: Add white space at the end of this line to avoid merge conflicts

**Comment by christophavila** [comment]
Added white space at the end of line 96 of espanol/canudos.md to avoid merge conflict

**Comment by christophavila** [comment]
I don´t have any feedback. I will close the pull request and create another one.
